src/mmwtex.c

```diff
@@ -59,7 +59,7 @@ pntrString_def(g_mathboxUser); // User name vs. mathbox #
 // in a math string in a comment to be removed for HTML output.
 #define CLOSING_PUNCTUATION ".,;)?!:]'\"_-"
 
-// Tex output file
+// LaTeX output file
```

Comment:
nitpick: Does metamath-exe actually produce LaTeX (as compared to TeX) output? I would not be surprised if it is plain TeX.

Reply:
`metamath-exe` produces LaTeX code that is not TeX-compatible, since it outputs "\documentclass" for instance, and uses several packages.